lametro/templates/snippets/related_object_status_tag.html

```diff
@@ -0,0 +1,4 @@
+{% load wagtailadmin_tags %}
```

Comment:
More or less stolen from the Wagtail source code, to make the status icon in the board member details snippet list link to the person detail page.